CONTRIBUTING.md

```diff
@@ -54,6 +54,11 @@ Try to keep lines of code no longer than 160 characters wide. This isn't a stric
 
 Try to keep your coding style in line with the existing code. It might not exactly match your preferred style but it's better to keep things consistent. StyleCop compliance is enforced through analysis on each build and the treatment of warnings as errors. Any StyleCop settings changes or suppressions must be clearly justified.
 
+## String Formatting
+
+Unless there is a compelling reason not to, for example when serializing data to be parsed later, use the current culture when formatting strings. When string
+formatting methods have an overload that implicitly uses the current culture, opt to use it instead of specifying the culture explicitly.

```

Comment:
> We like a 160 character limit, as set in .editorconfig

That's true, but this is an artificial break in the middle of a literal, and that's not something we'd do in C# either. In MarkDown it happens to be ignored in rendering but that's coincidental. If you look through the rest of the file, we don't add line breaks like these anywhere, so this would be the first incidence of it.